android-sdk/tasks/main.yml

```diff
@@ -1,9 +1,29 @@
 ---
 ## setup default template directories
 -
-  name: "Set default template directory"
-  set_fact: buildfarm_templates_dir="/opt/rhmap/{{ rhmap_version }}/templates/buildfarm"
-  when: buildfarm_templates_dir is not defined
+  name: "Check and include productised release image versions"
+  include_vars: ../shared_files/{{ rhmap_version }}.yml
+  when: rhmap_version is defined
+
+- name: "Include var file for community resources"
+  include_vars: vars/buildfarm.yml
+  when: rhmap_version is not defined
+
+-
+  name: "Read android sdk definition"
+  set_fact:
+    android_sdk_image_def: "{{ deployments | selectattr('name', 'match', '^android-sdk$') | map(attribute='containers') | list }}"
+
+-
+  name: "Get image version"
+  set_fact:
+    android_sdk_image: "{{ android_sdk_image_def[0][0].image }}"
+
+-
+  name: "Update local android-sdk-persistent template with image version and tag"
+  template:
+    src: android-sdk-persistent-template.j2
```

Comment:
Fails here, log:
`TASK [digger-installer/android-sdk : Update local android-sdk-persistent template with image version and tag] ***
fatal: [10.8.174.221]: FAILED! => {"changed": false, "failed": true, "msg": "Unable to find 'android-sdk-persistent-template.j2' in expected paths."}`